library/std/src/ffi/os_str.rs

```diff
@@ -265,6 +266,43 @@ impl OsString {
         self.inner.reserve(additional)
     }
 
+    /// Tries to reserve capacity for at least `additional` more elements to be inserted
```

Comment:
What is "elements" in this context? I don't think OsString supports inserting "elements".

OsStr::len and OsString::with_capacity have some more careful wording that I think try_reserve will need to adopt too.

- https://doc.rust-lang.org/1.57.0/std/ffi/struct.OsStr.html#method.len
- https://doc.rust-lang.org/1.57.0/std/ffi/struct.OsString.html#method.with_capacity